### PR TITLE
Consistent cmake option naming

### PR DESCRIPTION
**Pull request by jpfr**
@Stasik0 @FlorianPalm 
Are you ok with the renaming? It should be easy to adapt local build scripts.

### CODE REVIEW

**Comment by jpfr** [comment]
@ichrispa
I hope the cmake option naming won't break anything on your end.

If nobody objects, this will be merged tomorrow.

**Comment by Stasik0** [comment]
@jpfr probably a good time to make CI compile the examples aswell

File: doc/mainpage.dox [hunk 1]
```diff
@@ -1,110 +0,0 @@
-/**
```
Comment (Stasik0) [inline doc/mainpage.dox]: is this one beein copied somewhere else?

**Comment by Stasik0** [comment]
oh no, it cannot be merged anymore

**Comment by jpfr** [comment]
Master was merged. So the PR is currently read.
Only coveralls complains that the test coverage went down a bit.

**Pull request merged**